crates/bevy_input/src/keyboard.rs

```diff
@@ -31,206 +46,365 @@ pub fn keyboard_input_system(
     }
 }
 
-/// The key code of a keyboard input.
+/// The key code of a [`KeyboardInput`](crate::keyboard::KeyboardInput).
+///
+/// ## Usage
+///
+/// It is used as the generic `T` value of an [`Input`](crate::Input) to create a `bevy`
+/// resource. The resource stores the data of the buttons of a keyboard and can be accessed
+/// inside of a system.
+///
+/// ## Updating
+///
+/// The resource is updated inside of the [`keyboard_input_system`](crate::keyboard::keyboard_input_system).
 #[derive(Debug, Hash, Ord, PartialOrd, PartialEq, Eq, Clone, Copy)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 #[repr(u32)]
 pub enum KeyCode {
-    /// The '1' key over the letters.
+    /// The `1` key over the letters.
     Key1,
-    /// The '2' key over the letters.
+    /// The `2` key over the letters.
     Key2,
-    /// The '3' key over the letters.
+    /// The `3` key over the letters.
     Key3,
-    /// The '4' key over the letters.
+    /// The `4` key over the letters.
     Key4,
-    /// The '5' key over the letters.
+    /// The `5` key over the letters.
     Key5,
-    /// The '6' key over the letters.
+    /// The `6` key over the letters.
     Key6,
-    /// The '7' key over the letters.
+    /// The `7` key over the letters.
     Key7,
-    /// The '8' key over the letters.
+    /// The `8` key over the letters.
     Key8,
-    /// The '9' key over the letters.
+    /// The `9` key over the letters.
     Key9,
-    /// The '0' key over the 'O' and 'P' keys.
+    /// The `0` key over the `O` and `P` keys.
```

Comment:
```suggestion
    /// The `0` key over the letters.
```
If they can find 1 through 9 they can probably find 0 :wink:, and this won't be true for all keyboard layouts.

Reply:
Yep, "over the letters" is better.

Reply:
Even "over the letters" isn't true for some keyboards. Some people using tiny keyboards have the number keys on a different layer entirely. Or even ergonomic split keyboards might have the numbers somewhere else.

Reply:
Did some 'research' on google images.

Every Split keyboard I could find that had number keys, had them positioned over the letters.
40% keyboards indeed have the numbers on the top row on another layer, but I believe that someone using a niche keyboard will realize what is meant by 'over the letters'.

Reply:
Yeah, I should have been more specific, my friend has a split keyboard with blank keys and custom layers so the numbers could be anywhere but that's not really related to the fact that it's a split keyboard.

Either way, I still think that mentioning the position of the number keys relative to the letter is unnecessary, but I won't block the PR on that.

Reply:
I think the critical idea is "not numpad", which should be conveyed somehow.